docs/current_docs/cookbook/cookbook.mdx

```diff
@@ -613,6 +738,108 @@ This is an example of [post-call filtering](../api/fs-filters.mdx#post-call-filt
     </TabItem>
     </Tabs>
 
+### Mount a local or remote file in a container
```

Comment:
any reason why mounting a file and directory are so apart from each other from the cookbook? If I land into any of these sections by any chance, it'd be nice to know about the other option by scanning the docs. 

This is not a blocker to merge, just curious about the order here.